Directory.Build.props

```diff
@@ -1,20 +1,25 @@
 <Project>
-  <Import Project=".\scripts\sdk\Sdk.props" />
```

Comment:
❔ Is this file still used elsewhere? If not, was it deleted?

Reply:
This is a part of the current arcade build scripts. I plan to review the whole build pipeline once this is merged.
Some of these implementations can surely be removed when the remaining tasks (such as plugin loading, packaging, etc.) are resolved.